src/components/StylesFromLiteralsService.js

```diff
@@ -219,47 +228,69 @@ goog.provide('ga_stylesfromliterals_service');
         return this.defaultStyle;
       };
 
+      OlStyleForPropertyValue.prototype.setOlText_ = function(olStyle,
+        labelProperty, labelTemplate, properties) {
+        var text;
+        properties = properties || [];
+        if (labelProperty) {
+          text = properties[labelProperty];
+          if (text !== undefined && text !== null) {
```

Comment:
The text value is already tested further, no need to do it twice.

Reply:
May be I see when it could be useful to transofrm the value before the further test.
Are you able to provide a test when a propety value is 0 ? This is typically this kind of thing that we forget  in a refactoring.